pkg/sink/kafka/v2/admin.go

```diff
@@ -230,6 +230,9 @@ func (a *admin) CreateTopic(
 }
 
 func (a *admin) Close() {
```

Comment:
Looks like there is no IO block in the `Close`.

Reply:
https://github.com/segmentio/kafka-go/blob/main/writer.go#L1016

When close the writer, it tries to flush all queued batches, and retry for `MaxAttempts`, this is time-consuming.

After setting the `MaxAttempts` to 2 and `WriteBackOffMin` to `10ms`, still need around 5 ~ 15s.